dev-tools/install.sh

```diff
@@ -46,3 +46,4 @@ python3 -m venv .venv
 source .venv/bin/activate
 pip3 install --upgrade pip
 pip3 install -e .[dev]
+pip3 install django-cors-headers
```

Comment:
Can you add this to the `install_requires` of `setup.py` instead of the `install.sh`?